apps/web/.env.example

```diff
@@ -13,4 +13,6 @@ GOOGLE_CLIENT_SECRET=
 
 REDIS_URL=redis://localhost:6379
 ABBY_FROM_EMAIL=no-reply@tryabby.dev
-GITHUB_OAUTH_TOKEN=
\ No newline at end of file
+GITHUB_OAUTH_TOKEN=
+
+PLAUSIBLE_DOMAIN=tryabby.com
```

Comment:
It would be nice if you could add this to the `schema.mjs` file and then access the env vars in a typesafe way rather than `process.env.SOMETHING`